modules/rest-api/pages/rbac.adoc

```diff
@@ -618,6 +619,130 @@ An attempt to add a user to one or more non-existent groups fails with `400 Bad
 
 The creation of an already existing group or local or external user succeeds with `200 OK`: the user is recreated with the newly specified role-assignments and group-memberships; and the group is recreated with the newly specified role-assignments and, optionally, a newly specified mapping.
 
```

Comment:
@timofey-barmin should review the correctness of the rbac endpoints here.

Added REST API documentation for the backup endpoint. [Preview here.](https://preview.docs-test.couchbase.com/doc-11199-backup-restore-users-and-groups/server/7.6/rest-api/rbac.html#backup-and-restore-users-and-groups)